setup.py

```diff
@@ -57,7 +57,7 @@ def find_packages(path, base=""):
 ]
 
 setup(name="registryaggregator",
-      version="0.6.5",
+      version="0.6.6",
```

Comment:
Looks like setup.py also needs a 'six' dependency, but GitHub won't let me suggest in an unchanged bit unfortunately!